docker-compose.yml

```diff
@@ -81,7 +81,7 @@ services:
       dockerfile: Dockerfile
       target: dev
     image: hyku_addons/dev:latest
-    command: bash -c "rm -f spec/internal_test_hyku/tmp/pids/server.pid && bundle exec rails server -b 0.0.0.0"
+    command: bash -c "rm -f spec/internal_test_hyku/tmp/pids/server.pid && bundle install && bundle exec rails server -b 0.0.0.0"
```

Comment:
By having this here (and in the sidekiq container as well), we do not have to build the container when a Gem is added/removed/updated. The actual `bundle install` takes a few seconds when there is nothing to change, so it doesn't add much to the application boot time.